crates/matrix-sdk-ui/src/timeline/pinned_events_loader.rs

```diff
@@ -74,6 +77,7 @@ impl PinnedEventsLoader {
             let provider = self.room.clone();
             async move {
                 match provider.load_event(&event_id, request_config).await {
+                match provider.load_event_with_relations(&event_id, request_config).await {
```

Comment:
uh?

Reply:
I think it was a rebase hiccup, the code is not there later 😅 .